asv_bench/benchmarks/attrs_caching.py

```diff
@@ -15,7 +15,7 @@ def setup(self):
         self.cur_index = self.df.index
 
     def time_get_index(self):
-        self.foo = self.df.index
+        self.df.index = self.df.index
```

Comment:
🤔 not sure about reassigning to itself, how about assigning to e.g. `self.placeholder = self.df.index`?

Reply:
@MarcoGorelli this was my initial inclination, but I had changed it due to the feedback from @mzeitlin11 above

Reply:
Sorry for any confusion - what I meant in https://github.com/pandas-dev/pandas/pull/49379#discussion_r1008609473 was to just not reassign back at all. Left a more specific code suggestion too but accidentally left that as a pending review comment so it never got made :(